project/src/components/index.ts

```diff
@@ -2,3 +2,9 @@ export {default as App} from './app/app';
 export {default as FilmCard} from './film-card/film-card';
 export {default as Header} from './header/header';
 export {default as Footer} from './footer/footer';
+export {default as PrivateRoute} from './private-route/private-route';
+export {default as Poster} from './poster/poster';
+export {default as PosterDescription} from './poster-description/poster-description';
+export {default as GenreMenu} from './genre-menu/genre-menu';
+export {default as FilmMenu} from './film-menu/film-menu';
+export {default as RatingSelect} from './rating-select/rating-select';
```

Comment:
```suggestion
export * from './film-card/film-card';
```

так тоже должно работать)